### PR TITLE
Box select debug

### LINKED ISSUE #327: Direction of drag for selecting multiple images
Hi! I'm really enjoying the multiple image select in the latest release. I noticed that I seem to be able to select only by dragging from lower-right to upper-left, but no other direction (i.e. not from NE>SW or SW>NE or NW>SE). I was wondering if we could adapt the selection math to address this?

image

Potentially related to #29 and #244 -- thanks!

### PR DESCRIPTION
**Pull request by sashadev-sky**
Fixes #327  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

====================

changes: 
  - updated `Map.__newLayerPointToLatLng.js`. `L.Map.prototype._getNewTopLeft.call(...)` was not a valid method, I think `L.Map.prototype._getTopLeftPoint.call(...)`was meant. Still not used but could be useful!
  - `layerPointToLatLng` switched to `containerPointToLatLng` and a call to `map._latLngBoundsToNewLayerBounds()` added ;
  - pixel origin html file for future debugging
- version bump `0.6.3`

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@jywarren found an explanation in the Leaflet docs for why on panning it broke but not zoom:
image

**Comment by jywarren** [comment]
Aha! Awesome work! Is this ready for a merge? Thanks Sasha!!

**Comment by sashadev-sky** [comment]
@jywarren ok its good to go! Just see PR description and if you confirm you can merge!

File: debug/pixelorigin.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```
Comment (jywarren) [inline debug/pixelorigin.html]: Ah I think this file isn't supposed to be here still?

Comment (sashadev-sky) [reply debug/pixelorigin.html]: I kinda liked it enough to keep it but I can delete it if you want! This was the debug test file I was referring to. It helped me understand pixel origin and we can try getting it integrated with our UI too

**Comment by sashadev-sky** [comment]
bumped the version!

**Comment by sashadev-sky** [comment]
@rexagod can I get a review here if you have a minute :)

**Comment by jywarren** [comment]
Great! Let's release a new version!!!

**Comment by sashadev-sky** [comment]
@jywarren ok ill rebase and merge firefox fix then publish to npm 👍

**Pull request merged**